scale/recipe/configuration/data/recipe_data.py

```diff
@@ -123,6 +123,18 @@ def get_input_file_ids(self):
                 file_ids.update(data_input['file_ids'])
         return file_ids
 
+    def get_input_file_name(self, file_id):
+        """Returns the file name associated with the given file id.
+
+        :param file_id: The id used to find an associated name
+        :type file_id: int
+        :returns: The file name associated with the provided file id
+        :rtype: str
+        """
+
+        selected_input = [data['name'] for data in self.data_dict['input_data'] if data['file_id'] == file_id]
+        return selected_input[0]
```

Comment:
I believe this will break for recipes that have non-file input data. Needs to be more robust.